lib/validators/options_processor/options/no_empty.js

```diff
@@ -26,9 +26,14 @@ class NoEmpty {
     if (typeof input === 'string') {
       isMergeable = input.trim().length !== 0
     } else if (Array.isArray(input)) {
-      isMergeable = input.length !== 0
+      isMergeable = true
     } else {
-      throw new Error(UNKNOWN_INPUT_TYPE_ERROR)
+      if (input == null) {
```

Comment:
I think we use lodash `_.isUndefined` would vet the input better to see if its null/undefined

Reply:
@shine2lay hmm I do not think `_.isUndefined` is enough, I still have to use `== null`